gazebo_plugins/CMakeLists.txt

```diff
@@ -87,60 +86,60 @@ endif()
 
 catkin_package(
   INCLUDE_DIRS include
-  LIBRARIES
-  vision_reconfigure
-  gazebo_ros_utils
-  gazebo_ros_camera_utils
-  gazebo_ros_camera
-  gazebo_ros_multicamera
-  gazebo_ros_depth_camera
-  gazebo_ros_openni_kinect
-  gazebo_ros_gpu_laser
-  gazebo_ros_laser
-  gazebo_ros_block_laser
-  gazebo_ros_p3d
-  gazebo_ros_imu
-  gazebo_ros_f3d
+  LIBRARIES 
+  vision_reconfigure 
+  gazebo_ros_utils 
+  gazebo_ros_camera_utils 
+  gazebo_ros_camera 
+  gazebo_ros_multicamera 
+  gazebo_ros_depth_camera 
+  gazebo_ros_openni_kinect 
+  gazebo_ros_gpu_laser 
+  gazebo_ros_laser 
+  gazebo_ros_block_laser 
+  gazebo_ros_p3d 
+  gazebo_ros_imu 
+  gazebo_ros_f3d 
   gazebo_ros_ft_sensor
-  gazebo_ros_bumper
-  gazebo_ros_template
-  gazebo_ros_projector
-  gazebo_ros_prosilica
-  gazebo_ros_force
-  gazebo_ros_joint_trajectory
-  gazebo_ros_joint_state_publisher
+  gazebo_ros_bumper 
+  gazebo_ros_template 
+  gazebo_ros_projector 
+  gazebo_ros_prosilica 
+  gazebo_ros_force 
+  gazebo_ros_joint_trajectory 
+  gazebo_ros_joint_state_publisher 
   gazebo_ros_joint_pose_trajectory
   gazebo_ros_diff_drive
   gazebo_ros_tricycle_drive
   gazebo_ros_skid_steer_drive
   gazebo_ros_video
   gazebo_ros_planar_move
   gazebo_ros_vacuum_gripper
-
-  CATKIN_DEPENDS
-  message_generation
-  gazebo_msgs
-  roscpp
-  rospy
-  nodelet
-  angles
-  std_srvs
-  geometry_msgs
-  sensor_msgs
-  nav_msgs
-  urdf
-  tf
+  
+  CATKIN_DEPENDS 
+  message_generation 
+  gazebo_msgs 
+  roscpp 
+  rospy 
+  nodelet 
+  angles 
+  std_srvs 
+  geometry_msgs 
+  sensor_msgs 
+  nav_msgs 
+  urdf 
+  tf 
   tf2_ros
-  dynamic_reconfigure
-  rosgraph_msgs
-  trajectory_msgs
-  image_transport
+  dynamic_reconfigure 
+  rosgraph_msgs 
+  trajectory_msgs 
+  image_transport 
   rosconsole
   camera_info_manager
   std_msgs
-  DEPENDS
-    GAZEBO
-    SDFormat
+  DEPENDS 
```

Comment:
I think that what fixed the regression is really in this line although I just proceed with the git revert command.